### PR TITLE
Added Functionality to Valueinfofields & added "ResetValue" Button to  Progressbar

### PR DESCRIPTION
**Pull request by SergeyMOV**

### CODE REVIEW

File: components/ProgressBar.js [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+  const addFive = budget + addvalue + minusvalue + "€";
   const Budget = budget + "€";
   return (
     <Progressmain className="progressmain">
-      <Procentage1 value={managepercentage + "%"} />
-      <Procentage2 value={Budget && addFive} readonly />
-      <Progressbar value={progress} max="100"></Progressbar>
+      <Procentage1 ref={ref2} value={managepercentage + "%"} />
+      <Procentage2 ref={ref} value={Budget && addFive} />
+      <Restorebutton onClick={restoreClick}>
```
Comment (actyralf) [inline components/ProgressBar.js:29]: Why are you using `refs` for your values? Shouldn't this be state variables?
Furthermore: The percentage value can be calculated at runtime, so no need to have a separate variable for it.
Maybe we should have a look at your code together. I don't understand what these refs are used for.

File: pages/index.js [hunk 2]
```diff
@@ -47,18 +48,20 @@ export default function Home() {
       <Navmenu />
       <Valuefields
         addvalue={addvalue}
-        handlesubtractvalues={handleSubtractValues}
-        onChangeSubtract={handleSubtractValue}
+        minusvalue={minusvalue}
+        handleSubtractValues={handleSubtractValues1}
+        onChangeSubtract={handleSubtractValues}
         onChangeValue={handleAddValue}
```
Comment (actyralf) [inline pages/index.js:54]: Naming: You shouldn't have functions `handleSubtractValues` and `handleSubtractValues1`. It's not clear what's the difference between these functions.

**Review by actyralf — CHANGES_REQUESTED** [review]
Hey Sergey,

it's not clear to me what these `ref` variables are used for and why it's necessary to have them. Maybe we can have a joint look on your code tomorrow...

File: pages/index.js (hunk 2 above)
Comment (SergeyMOV) [reply pages/index.js:54]: Actually fixed!

File: components/ProgressBar.js (hunk 1 above)
Comment (SergeyMOV) [reply components/ProgressBar.js:29]: Ok, then we will have a look at my Code Tomorrow.

**Pull request merged**